pkg/server/util/api.go

```diff
@@ -13,7 +13,7 @@ import (
 var DefaultDashboardPort = ":38081"
 
 // DefaultAPIServerPort refers to the default port number of APIServer
-const DefaultAPIServerPort = ":8081"
+const DefaultAPIServerPort = ":38081"
```

Comment:
Why change this?

If you want to change the port, also change the `dev`config in proxy.ts .

Please also add a comment here.

Reply:
New version of Alilang takes the port at system start. Okay. changed the `dev` config. Thanks.